playwright.config.ts

```diff
@@ -0,0 +1,93 @@
+import type { PlaywrightTestConfig } from '@sand4rt/experimental-ct-web';
```

Comment:
please can you ensure this has been run through our ESLINT configuration and doesn't flag a bunch of red in vscode IDE like the `wdio` config file does? (sorry if you already have)